VoteCounter/src/GUI.cs

```diff
@@ -29,8 +29,48 @@ private void btnGenerateVoteCount_Click(object sender, EventArgs e)
             {
                 PostList postList = new PostList(url, startingPostNumber, endingPostNumber);
 
+                //make a List
+                List<Player> players = new List<Player>();
+
+                //set the players
+                foreach (DataGridViewRow row in listPlayers.Rows)
+                {
+                    List<string> nicks = new List<string>();
+                    string name = "";
```

Comment:
Rather than this, I think I'd create a new Player object and set its values as we go through the loop. It also means we can bail out much earlier if the name is invalid.